redirects.json

```diff
@@ -173,6 +171,35 @@
     {
       "source": "/:locale/docs/guides/timers-in-node/",
       "destination": "/:locale/learn/asynchronous-work/discover-javascript-timers"
+    },
+    {
+      "source": "/:locale/docs/es6",
+      "destination": "/:locale/learn/getting-started/ecmascript-2015-es6-and-beyond"
+    },
+    {
+      "source": "/:locale/docs",
+      "destination": "https://nodejs.org/docs/latest/api/"
+    },
+    {
+      "source": "/:locale/docs/guides",
+      "destination": "/:locale/guides"
+    },
+    {
+      "source": "/:locale/docs/guides/:path*",
+      "destination": "/:locale/guides/:path*"
+    },
+    {
+      "source": "/:locale/get-involved",
+      "destination": "/:locale/about/get-involved"
+    },
+    {
+      "source": "/:locale/get-involved/:path*",
+      "destination": "/:locale/about/get-involved/:path*"
+    },
+    {
+      "source": "/:locale/learn",
+      "destination": "/:locale/learn/getting-started/introduction-to-nodejs"
     }
-  ]
+  ],
+  "internal": []
```

Comment:
with internal now empty, how relevant is this code within https://github.com/nodejs/nodejs.org/blob/8ef12a4728c4553ad7a1bbfc50fad88d6f2f107b/next.rewrites.mjs#L41